apps/backoffice-v2/src/lib/blocks/hooks/useBankAccountVerificationBlock/useBankAccountVerificationBlock.tsx

```diff
@@ -0,0 +1,76 @@
+import { useCallback, useMemo } from 'react';
+
+import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+
+export const useBankAccountVerificationBlock = ({ pluginsOutput }) => {
+  const getCell = useCallback(() => {
+    if (
+      Object.keys(pluginsOutput?.bankAccountVerification?.data.clientResponsePayload ?? {}).length
+    ) {
+      const data = {
+        ...pluginsOutput.bankAccountVerification.data.responseHeader.overallResponse,
+        decisionElements:
+          pluginsOutput.bankAccountVerification.data.clientResponsePayload.decisionElements,
+        orchestrationDecisions:
+          pluginsOutput.bankAccountVerification.data.clientResponsePayload.orchestrationDecisions,
+      };
+
+      return {
+        id: 'nested-details',
+        type: 'details',
+        hideSeparator: true,
+        value: {
+          data: Object.entries(data)
+            ?.filter(([property]) => !['tenantID', 'clientReferenceId'].includes(property))
+            .map(([title, value]) => ({
+              title,
+              value,
+            })),
+        },
+        props: {
+          config: {
+            sort: { predefinedOrder: ['decision', 'decisionText'] },
+          },
+        },
+      } satisfies Extract<
+        Parameters<ReturnType<typeof createBlocksTyped>['addCell']>[0],
+        {
+          type: 'details';
+        }
+      >;
+    }
+  }, [pluginsOutput]);
```

Comment:
You should probably use exact nested object as dependency to avoid rerenders on every change of pluginsOutputs (pluginsOutput.bankAccountVerification.data)